proto/types/blockchain.proto

```diff
@@ -30,6 +30,7 @@ message BlockHeaderCore {
   bytes deltaStateDigest = 6;
   bytes receiptRoot = 7;
   bytes logsBloom = 8;
+  int64 numActions = 9;
```

Comment:
This line interested to me.
I found 4 definitions of numActions in blockchain.proto.
3 of they are `int64`, where the other is `uint64` defined in AccountMeta.
Could we use all them as one type?

Reply:
Let’s use uint64. For those are int64, we unfortunately have to keep it to avoid breaking change

Reply:
Changed to uint64.